### PR TITLE
chore(test): move expected_conditions_spec off of the control flow

### PR DESCRIPTION
**Pull request by CrispusDH**
@cnishina , could you take a look at this PR, please

### CODE REVIEW

**Comment by CrispusDH** [comment]
Reference #4995

**Comment by cnishina** [comment]
The travis test failing is because of the ciNg2 test. I hope to have this resolved. It might require one more rebase. I'll keep you posted.

File: spec/basic/expected_conditions_spec.js [hunk 1]
```diff
@@ -1,255 +1,250 @@
...
-    var validityOfTitle = EC.titleIs('My AngularJS App');
-
-    expect(EC.and(presenceOfValidElement, validityOfTitle).call()).toBe(true);
+const EC = protractor.ExpectedConditions;
+
+describe('expected conditions', () => {
+  beforeEach(async () => {
+    await browser.get('index.html#/form');
+  });
+
```
Comment (cnishina) [inline spec/basic/expected_conditions_spec.js]: Please remove spaces.

File: spec/basic/expected_conditions_spec.js [hunk 2]
```diff
@@ -1,255 +1,250 @@
...
+  it('should have alertIsPresent', async () => {
+    const alertIsPresent = EC.alertIsPresent();
+    expect(await alertIsPresent.call()).toBe(false);
+    
+    const alertButton = $('#alertbutton');
+    await alertButton.click();
+    await browser.wait(protractor.ExpectedConditions.alertIsPresent(), 5000);
+    await browser.switchTo().alert().accept();
+  });
+
```
Comment (cnishina) [inline spec/basic/expected_conditions_spec.js]: Here too. This is introducing a full file change.

File: spec/basic/expected_conditions_spec.js [hunk 3]
```diff
@@ -1,255 +1,250 @@
...
+    const alertButton = $('#alertbutton');
+    await alertButton.click();
+    await browser.wait(protractor.ExpectedConditions.alertIsPresent(), 5000);
+    await browser.switchTo().alert().accept();
+  });
+  
+  it('should have presenceOf', async () => {
+    const presenceOfInvalid = EC.presenceOf($('#INVALID'));
+    const presenceOfHideable = EC.presenceOf($('#shower'));
+
```
Comment (cnishina) [inline spec/basic/expected_conditions_spec.js]: Here and everywhere. Thanks!

File: spec/basic/expected_conditions_spec.js (hunk 1 above)
Comment (cnishina) [reply spec/basic/expected_conditions_spec.js]: Hrrmmm... it looks like this doesn't show it. If you look at your diff, you will see what I'm seeing. https://github.com/angular/protractor/pull/5013/files

**Comment by cnishina** [comment]
Thank you for cleaning all those spaces up!

**Comment by CrispusDH** [comment]
Thank you for giving me possibility to contribute :)

**Pull request merged**